terms.html

```diff
@@ -62,10 +70,11 @@
   <dt><dfn data-lt="holders|holder's|holders'">holder</dfn></dt>
   <dd>
 A role an <a>entity</a> might perform by possessing one or more
-<a>verifiable credentials</a> and generating <a>presentations</a> from them.
-A holder is usually, but not always, a <a>subject</a> of the <a>verifiable
-credentials</a> they are holding. Holders store their <a>credentials</a> in
-<a>credential repositories</a>.
+<a>verifiable credentials</a> and generating <a>verifiable presentations</a>,
+both from such <a>verifiable credentials</a> and <a>claims</a> it authors itself.
```

Comment:
We currently don't have consensus on treating arbitrary properties in a VP as claims, so all of the claims come from VCs today. 
```suggestion
<a>verifiable credentials</a> and generating <a>presentations</a> from them.
A holder is usually, but not always, a <a>subject</a> of the <a>verifiable
credentials</a> they are holding. Holders store their <a> verifiable credentials</a> in
<a>credential repositories</a>.
```